request-interfaces.go

```diff
@@ -74,6 +74,10 @@ type StatVFSFileCmder interface {
 // FileLister should return an object that fulfils the ListerAt interface
 // Note in cases of an error, the error text will be sent to the client.
 // Called for Methods: List, Stat, Readlink
+// For Readlink, Filelist is limited to os.FileInfo response. For a more
```

Comment:
I might recommend a paragraph break here. The added whitespace makes the separation of the purpose of the two comments more clear.

Also, I know the existing text looks like it’s using a 80 column break, each of the three existing lines is a distinct individual semantic statement. In Golang, you don’t want to be breaking up lines just because you’re fitting a column width, but rather because they’re natural breaks. So more https://sembr.org/ 

So:
```
// For Readlink, Filelist is limited to os.FileInfo response.
// For more sophisticated Readlink response, please implement ReadlinkFileLister.
// If ReadlinkFileLister is implemented, its Readlink method is called for method Readlink instead of calling Filelist.
```

I’d probably also suggest being more explicit about what “sophisticated” is supposed to mean. So maybe even better:
```
// Since Filelist returns an `os.FileInfo`, this can make it non-ideal for impelmenting Readlink.
// This is because the `Name` receiver method defined by that interface defines that it should only return the base name.
// However, `Readlink` is required to be capable of returning essentially any arbitrary valid path relative or absolute.
// In order to implement this more expressive requirement, implement `ReadlinkFileLister`.
```

Reply:
I will just copy youre suggestion here